nipype/interfaces/base/support.py

```diff
@@ -215,34 +196,68 @@ class InterfaceResult(object):
         * stderr : Any error messages output from running ``cmdline``.
         * returncode : The code returned from running the ``cmdline``.
 
+
+    >>> rt = InterfaceRuntime(cmdline='/bin/echo', returncode=0)
+    >>> result = InterfaceResult(interface='CommandLine',
+    ...                          runtime=rt)
+    >>> import pickle
+    >>> pickleds = pickle.dumps(result)
+    >>> newresult = pickle.loads(pickleds)
+    >>> newresult  # doctest: +ELLIPSIS
+    InterfaceResult(interface='CommandLine', runtime=\
+Bunch(cmdline='/bin/echo', returncode=0), version=...)
+
     """
+    __slots__ = ['interface', 'runtime', 'inputs', 'outputs', 'provenance']
+    version = __version__
```

Comment:
I think the goal of the `version` attribute was to invalidate results loaded from an incompatible format, not record the nipype version.

Reply:
MMM so this is why we had 2.0 here? was that the pickle protocol version?

Reply:
@satra Perhaps you can clarify? I interpreted as the `InterfaceResult` API version.